### ISSUE TITLE
[Kubernetes Gradle Plugin] Image build is not skipped if `build` > `skip` is set to `true`

### ISSUE THREAD

**Issue by rohanKanojia** [eclipse/jkube#1197]
### Description
While investigating https://github.com/eclipse/jkube/issues/1195, I noticed in gradle plugins we're not considering `skip` field in build configuration. 

If I add a `<skip>true</skip>` field in image build configuration like this:

```xml
      <plugin>
        <groupId>org.eclipse.jkube</groupId>
        <artifactId>kubernetes-maven-plugin</artifactId>
        <version>${jkube.version}</version>
        <configuration>
          <images>
            <image>
              <name>${project.groupId}/${project.artifactId}</name>
              <build>
                <from>quay.io/jkube/jkube-java-binary-s2i:${jkube.image.version}</from>
                <skip>true</skip>
```
I can see image build is skipped on executing `k8s:build`:
```
[INFO] Building Eclipse JKube :: Quickstarts :: Maven :: Micronaut customized Image 1.5.1
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- kubernetes-maven-plugin:1.5.1:build (default-cli) @ micronaut-customized-image ---
[INFO] k8s: Running in Kubernetes mode
[INFO] k8s: Building Docker image in Kubernetes mode
[INFO] k8s: [org.eclipse.jkube.quickstarts.maven/micronaut-customized-image:latest] : Skipped building
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.163 s
[INFO] Finished at: 2022-01-06T22:14:19+05:30
[INFO] ------------------------------------------------------------------------
```

However, If I add `skip=true` field in quickstarts/gradle/micronaut-customized-image, image building is not skipped:

```groovy
kubernetes {
    images {
        image {
            name = "%g/%a:${project.version}"
            build {
                skip = true
               // ...
            }
        }
    }
```

We need to add filter images like done in AbstractDockerMojo in KubernetesBuildTask also:
We can filter out image configurations here which don't have `skip` enabled like done for build goal:
https://github.com/eclipse/jkube/blob/b66213473b17a4d4c25096fc02055ce33197797b/kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractDockerMojo.java#L485-L489
### Info

* Eclipse JKube version : 1.5.1

* Kubernetes / Red Hat OpenShift setup and version : N/A

* If it's a bug, how to reproduce : go to quickstarts/gradle/micronaut-customized-image and do `gradle k8sBuild` after setting `skip = true` in `build { .. }` section

**Comment by manusa** [comment]
TL; DR  Gradle build tasks don't account for Image model configuration skip field.
